civihr_employee_portal/civihr_employee_portal.install

```diff
@@ -317,10 +318,29 @@ function civihr_employee_portal_update_7020() {
   _civihr_employee_portal_create_default_reports_configuration();
 }
 
+/**
+ * Fixes contact images that have multiple 'photo' query params.
+ */
+function civihr_employee_portal_update_7021() {
+  civicrm_initialize();
+  $params = ['return' => ['image_URL']];
+  $contacts = civicrm_api3('Contact', 'get', $params)['values'];
+
+  foreach ($contacts as $contact) {
+    $imageUrl = $contact['image_URL'];
+    $dedupedImageUrl = UrlHelper::dedupeUrlQueryParams($imageUrl);
```

Comment:
This will not only photo duplicates but any duplicate param. Is this really want we want? Wouldn't it be a good idea if we could specify which param we want to be fixed?

Reply:
There isn't really a standard around this area. I thought using arrays like `foo[]=bar&foo[]=foo` was the norm, but it seems to depend on which language you're using. For PHP anyway, if you use `foo=bar&foo=foo` it will use the last value for `foo`. This is what led to the bug this ticket is fixing.

Since the environment is PHP I think removing all duplicate params would be practical, but if you think it's best to only target photo I can update the upgrader and add a new method to the `UrlHelper` like `dedupeSpecificUrlQueryParam`

Reply:
Since now the helper can handle arrays, I think it's safe to keep this as is now.